src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -95,6 +96,10 @@ public FeatureOnOffOptions()
             FeatureName, "NavigateToDecompiledSources", defaultValue: true,
             storageLocation: new RoamingProfileStorageLocation("TextEditor.NavigateToDecompiledSources"));
 
+        public static readonly Option2<bool> AlwaysUseDefaultSymbolServers = new(
+            FeatureName, "AlwaysUseDefaultSymbolServers", defaultValue: true,
+            storageLocation: new RoamingProfileStorageLocation("TextEditor.AlwaysUseDefaultSymbolServers"));
```

Comment:
You can now remove both of these from here since they are defined in the -Storage class now.